source/index.ts

```diff
@@ -15,8 +16,11 @@ let backtraceClient: BacktraceClient;
  * Initalize Backtrace Client and Backtrace node integration
  * @param configuration Backtrace configuration
  */
-export function initialize(configuration: BacktraceClientOptions): BacktraceClient {
+export function initialize(
+  configuration: BacktraceClientOptions,
+): BacktraceClient {
   backtraceClient = new BacktraceClient(configuration);
+  backtraceMetric(configuration); // ping metrics. Called on initialization, which ~should~(?) be called on every navigation.
```

Comment:
shouldn't it belong to the backtrace client instance?

Reply:
Yes, updated to be inside the BacktraceClient